### DISCUSSION TITLE
help me

### DISCUSSION THREAD

**Discussion by iproazad** [N0-0NE-Dev/NoFasel#27] [Ideas]
can you help me i want add link video but after 24h its chang and link dont work

**Comment by abadawe-dev** [comment]
Message me on [telegram](https://t.me/AhmedBadawe147).

**Reply by iproazad** [reply]
![Screenshot_20230226_020141_org telegram messenger](https://user-images.githubusercontent.com/125834441/221383537-71840698-b759-42dc-8b33-19127f7bd286.jpg)

**Reply by abadawe-dev** [reply]
What's your telegram username?

**Reply by iproazad** [reply]
ipazad
